calcom/cal.com
# feat: managed event types v2

## What does this PR do?

Implements Managed Event Types v2 where the event owner/team admin can discretely select which field to lock or unlock in a managed event type.

<kbd><img src="https://github.com/calcom/cal.com/assets/467258/28379efe-18e6-4782-a0b7-63ccda794965" /></kbd>

<kbd><img src="https://github.com/calcom/cal.com/assets/467258/657e8a63-3519-4526-af55-b86bf9bdb889" /></kbd>

<kbd><img src="https://github.com/calcom/cal.com/assets/467258/4e9c4149-7a9c-42a0-b600-08c3631aeac4" /></kbd>

<kbd><img src="https://github.com/calcom/cal.com/assets/467258/c9edc1f6-ae58-4667-b4f9-d3b3a37b531d" /></kbd>

@ciaranha just FYI, went for the badge with title and the switch as it was the one that can be universally applied to any type of field, whether is a text or a setting toggle.

Talked to @PeerRich and he instructed me to tag you @CarinaWolli for this in order to have you take over MET.

Pending tasks: 


## Type of change


## How should this be tested?

Create a team and then a managed event type. Upon getting to the event type settings page, see the different toggles to lock/unlock them.
